concordium-node/src/configuration.rs

```diff
@@ -746,19 +749,19 @@ pub struct MacOsConfig {
 #[structopt(about = "Concordium P2P node.")]
 pub struct Config {
     #[structopt(flatten)]
-    pub common:       CommonConfig,
+    pub common: CommonConfig,
     #[cfg(feature = "instrumentation")]
     #[structopt(flatten)]
-    pub prometheus:   PrometheusConfig,
+    pub prometheus: PrometheusConfig,
     #[structopt(flatten)]
-    pub connection:   ConnectionConfig,
+    pub connection: ConnectionConfig,
     #[structopt(flatten)]
-    pub cli:          CliConfig,
+    pub cli: CliConfig,
     #[structopt(flatten)]
     pub bootstrapper: BootstrapperConfig,
     #[cfg(target_os = "macos")]
     #[structopt(flatten)]
-    pub macos:        MacOsConfig,
+    pub macos: MacOsConfig,
```

Comment:
This reformatting here suggests you have the wrong rust formatter. Please format with the correct nightly version as described in the README.